modules/Forum/module.php

```diff
@@ -59,7 +59,7 @@ public function __construct(Language $language, Language $forum_language, Pages
         EventHandler::registerEvent('newTopic',
             $this->_forum_language->get('forum', 'new_topic_hook_info'),
             [
-                'uuid' => $this->_language->get('admin', 'uuid'),
+                'user_id' => $this->_language->get('admin', 'user_id'),
```

Comment:
Why was this changed?

Reply:
Uuid wasn't provided when the event was triggered but the user id.

Reply:
As you can see on line 205 here https://github.com/NamelessMC/Nameless/blob/v2/modules/Forum/pages/forum/new_topic.php